transactions/FlowServiceAccount/scripts/check_if_payer_has_sufficient_balance.cdc

```diff
@@ -0,0 +1,8 @@
+import FlowFees from 0xFLOWFEESADDRESS
+
+pub fun main(payerAcct: Address, inclusionEffort: UFix64, maxExecutionEffort: UFix64): Bool {
+    let authAcct = getAuthAccount(payerAcct)
+    return FlowFees.verifyPayersBalanceForTransactionExecution(authAcct,
```

Comment:
I tried to write it like 
`
let result =  verifyPayersBalanceForTransactionExecution(...)
return result.variable
`
but got a runtime error saying that the result was already declared. Do you know what the reason is?

Reply:
I think this script should just return the same values as `verifyPayersBalanceForTransactionExecution`